src/NHPersistenceCompatibilityTests/6.1.0/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Reflection;
```

Comment:
Don't think we need it. It was removed from the other projects (the whole AssemblyInfo file)